openid-federation-1_0.xml

```diff
@@ -7420,24 +7436,123 @@ HTTP/1.1 302 Found
           <t>
             <?rfc subcompact="yes"?>
             <list style="symbols">
-              <t>Name: missing_trust_anchor</t>
+              <t>Name: invalid_request</t>
+              <t>Usage Location: authorization endpoint</t>
```

Comment:
This invalid_request error was meant for the federation endpoints, right?

Not sure why we have the authZ endpoint was usage location here.